config/initializers/carrier_wave.rb

```diff
@@ -0,0 +1,14 @@
+CarrierWave.configure do |config|
+  if Rails.env.staging? || Rails.env.production?
+    config.storage = :fog
+    config.fog_credentials = {
```

Comment:
I intend to keep it blank because my AWS account is not accessable now. I'll try it if I get my account back :-)